### DISCUSSION TITLE
posts/python%E5%9F%BA%E7%A1%80-%E7%BC%96%E7%A8%8B%E7%AB%A0%E6%B3%952/

### DISCUSSION THREAD

**Discussion by giscus** [roninro/hugo-theme-puppet#61] [General]
# posts/python%E5%9F%BA%E7%A1%80-%E7%BC%96%E7%A8%8B%E7%AB%A0%E6%B3%952/

[TOC] 今天继续分享一下自己的近期心得： python之禅 1Python 3.10.11 | packaged by Anaconda, Inc. | (main, Apr 20 2023, 18:56:50) [MSC v.1916 64 bit (AMD64)] on win32 2Type "help", "copyright", "credits" or "license" for more information. 3>>> import this 4The Zen of Python, by Tim Peters 5 6Beautiful is better than ugly.

https://zhenda-hub.github.io/posts/python%E5%9F%BA%E7%A1%80-%E7%BC%96%E7%A8%8B%E7%AB%A0%E6%B3%952/

**Comment by zhenda-hub** [comment]
good
